index.html

```diff
@@ -0,0 +1,461 @@
+                                </svg>                                
+                        </li>
+                        <li class="teams__list">
+                            <svg width="24" height="20" viewBox="0 0 24 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <path d="M24.0004 2.46548C23.1188 2.85141 22.1594 3.12917 21.1711 3.23735C22.1971 2.61879 22.9655 1.64156 23.332 0.489007C22.3692 1.07022 21.3145 1.47778 20.2146 1.6936C19.7549 1.19488 19.1989 0.797581 18.5813 0.526453C17.9636 0.255325 17.2976 0.116179 16.6247 0.117688C13.902 0.117688 11.7123 2.3573 11.7123 5.10564C11.7123 5.49158 11.7584 5.87752 11.8333 6.24884C7.7565 6.03248 4.12048 4.05601 1.7032 1.0299C1.26274 1.79335 1.03193 2.66261 1.03477 3.54727C1.03477 5.27814 1.902 6.80435 3.22444 7.70195C2.44511 7.6708 1.68402 7.45342 1.00308 7.06749V7.12889C1.00308 9.5527 2.69143 11.5613 4.94161 12.0233C4.51912 12.1347 4.08449 12.1916 3.64797 12.1929C3.32817 12.1929 3.02565 12.1607 2.72024 12.1168C3.34257 14.0933 5.15482 15.5289 7.3128 15.5757C5.62444 16.9177 3.50968 17.7071 1.2134 17.7071C0.801397 17.7071 0.421085 17.6925 0.0263672 17.6457C2.20452 19.0637 4.78891 19.8824 7.5721 19.8824C16.6074 19.8824 21.5514 12.2864 21.5514 5.69332C21.5514 5.47696 21.5514 5.2606 21.537 5.04424C22.4936 4.33377 23.332 3.45371 24.0004 2.46548Z" fill="#00A0C1"/>
+                                </svg>                                
+                        </li>
+                    </ul>
+                </div>
+                <div class="cardteams__item">
```

Comment:
### warning naming wrong

it will be good if you change `cardteams__item` to `card-teams__item`

Reply:
@IbrahimBagalwa ,Hi I did it !